rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -495,24 +407,14 @@ MasterGraph::reset()
     if(_output_thread.joinable())
         _output_thread.join();
     _ring_buffer.reset();
+    _sequence_start_framenum_vec.clear();
```

Comment:
If all the changes wrt to video was removed , why is this added / retained yet?

Reply:
Separate output routine for video and associated variables were removed. This is to be returned to user.